templates/image-streams-s390x.json

```diff
@@ -72,9 +52,9 @@
             "kind": "ImageStream",
             "apiVersion": "v1",
             "metadata": {
-                "name": "openjdk-11-rhel7",
+                "name": "openj9-11-rhel7",
```

Comment:
Couldn't `openj9-11-*` be added as alternatives to `openjdk-11-*` here, since at least the latter (unlike openjdk-8) does have a JIT for Z?  That would not contradict making openj9 the defaults for the `java` imagestream.

Reply:
Do you mean defining image streams named "openjdk-11-*" that are defined in terms of the OpenJ9 DockerImages?

(as always with OCP stuff) I'm not sure of the ramifications, and wary of setting an early precedent. Do you think that there are recipes, quickstarts etc out there that  could be potentially used without changes on Z if we did this?